### ISSUE TITLE
针对多GPu训练的问题

### ISSUE THREAD

**Issue by xiaoxusanheyi** [Cc-Hy/CMKD#29]
1. 执行了你们给出的多GPU训练的命令：
![image](https://user-images.githubusercontent.com/113592382/227776682-f43ce9cc-2cd5-4c66-b468-ff1fb857c132.png)
30个epoch 训练 还有10个深度训练  后面直接10次epoch  结果就是直接用测试集测试出来的最终结果吗   我拿到的这个结果是不是就可以和原文进行对比了。
2. 是不是没有用到验证集val呀

**Comment by Cc-Hy** [comment]
The above commands first use 30 epoch training with BEV feature distillation only to make the student model learn to generate reasonable BEV representations first and then use 10 epoch training with the complete training losses for detection optimization. And the second command will automatically run the eval process on the validation set. And the results given in the model zoo are obtained in the same way, you can compare them fairly.

**Comment by xiaoxusanheyi** [comment]
感谢作者的回复。
1.原文章中的表1给出的结果是测试集测出来的结果 ，而model zoo的结果是验证集上的结果，为什么差距这么大 ？所以说我们直接对比github model zoo的结果就行是吧，不需要与原文的结果做对比。
2.在代码中，我看到特征蒸馏loss的权重是32。为什么特征loss的权重是32 ？而response的权重为1，不是说权重太小的话损失函数就不起作用了吗？ 
3.如果说在添加一个改进的loss话，权重我该设置多少比较合理？
